.craft.yml

```
@@ -18,6 +18,7 @@ targets:
       drift:
       isar:
       link:
+      firebase:
```

```suggestion
      firebase_remote_config:
```

the folder name should also be changed to firebase_remote_config